### PR TITLE
Beato d10 7 5d 5e

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 1]
```diff
@@ -1,37 +1,37 @@
 [sha:174548517d159082577a6222d1f566275c277d40]{
 -- Page 10018, Offset 25709.
 -- こつこつと時計の針は進んでいく。
-The hands of the clock advances and ticks away.
+The hands on the clock advance.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:4]: The clock's hands

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 2]
```diff
@@ -106,12 +107,12 @@ What is she, a revolving kaleidoscope?
...
 -- Page 565, Offset 25730.
 -- 「……いや、何が起きたのかは分かるんだが……おまえ、黒い死者について考えごとしてるんじゃなかったのか？」
-"...No, I understand what happened, but... weren't you agonizing while thinking about that black Dead just now?"
+"...No, I understand what happened, but... weren't you agonizing while thinking about that shadowy Dead just a second ago?"
 }
 [sha:79152efc857736d5b3c1b939fa29a8c315dc0afb]{
 -- Page 566, Offset 25731.
 -- 「してたわよ。だから思い出したのっ！
-"I was. That's why I suddenly remembered!
+"I was. That's what made me suddenly remembered!
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:115]: remember

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 3]
```diff
@@ -216,22 +218,27 @@ I manage to retort back somehow before averting my gaze from Arcueid, who's in a
...
+// this rendition makes it a little hard to tell who is returning from a night out,
+// (students & couples or just students like in the jp) but I don't think it makes 
+// much of a difference either way.
+If it weren't for the serial murders, at this time we could expect to see young couples and students returning home from a night out.
 }
 [sha:dc4e537d204ceb836250f1a6c5611de16cc343f6]{
 -- Page 575, Offset 25754.
 -- 今では、ここで話しているのは自分とアルクェイドだけ。
-Now the only people who talk here are me and Arcueid.
+// is this ここで(話しているのは自分とアルクェイド)だけ
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:229]: I think it's this one?

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 4]
```diff
@@ -431,7 +439,7 @@ In that case, get up from the bench and come over here."
 [sha:c9ebcdde99e51d73e9420c81348f6c4af9c6169f]{
 -- Page 588, Offset 25796.
 -- 感情のない声で、アルクェイドはそう言った。
-Arcueid says that in a voice devoid of emotion.
+Arcueid's voice is cold.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:442]: I prefer "devoid of emotion" over "cold" here for no particular reason

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 5]
```diff
@@ -586,12 +594,12 @@ One step. And another.
...
 -- Page 598, Offset 25827.
 -- うつむいたまま、少しずつ。
-Little by little, with her head hanging down.
+With her head hanging low, little by little.
 }
 [sha:828ad6de62abc1e54909c7e8825bb308e7ab21a3]{
 -- Page 598, Offset 25828.
 -- 俺は指一本も動かせないまま、彼女の、<瑞々|みずみず>しくも肉感的な、唇を見つめている――――
-Unable to move even a single finger, I gaze upon her vibrantand enticing lips――――
+Unable to move even a single finger, I fix my eyes on her full, sensual lips――――
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:602]: cringe

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 6]
```diff
@@ -626,12 +634,12 @@ Her breath gets closer.
...
 -- Page 600, Offset 25835.
 -- 指先が触れる。
-Her fingers touch me.
+Her fingertips touch my body.
 }
 [sha:cb92bebb57217a91c2db532c43488e0e310643fd]{
 -- Page 600, Offset 25836.
 -- 肩口に、全身に、アルクェイドの重みを感じる。
-I feel the weight of her entire body on top of my shoulders.
+I feel her weight on of my shoulders; I feel it through my whole body.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:642]: drop of

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 7]
```diff
@@ -901,17 +909,17 @@ Arcueid's fingers won't let go.
...
 -- Page 613, Offset 25890.
 -- ―――いけない。それは、いけない。
-―――Not good. This is bad.
+―――Not good. This is not good.
 }
 [sha:7cc3f132250168a1c85e7d160b3853289fee5f49]{
 -- Page 613, Offset 25891.
 -- 誰にとって禁忌なのか。
-That must have been a taboo.
+I must have touched a taboo .
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:917]: extra space?

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 8]
```diff
@@ -976,7 +984,7 @@ There's no need for her to use brute strength. With her alluring voice alone...
 [sha:4e20551c7dd8642526fd4c74126a9545352a516a]{
 -- Page 617, Offset 25905.
 -- 肉体ごと、精神が溶かされる。
-My soul melts along with my entire body.
+My soul with my entire body melts.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:987]: revert imo

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 9]
```diff
@@ -1021,33 +1029,34 @@ Divine providence that could not be opposed.
...
 -- Page 620, Offset 25916.
 -- 山脈のようにとがった歯。
-Teeth spiked like a mountain range.
+Teeth, spiked, like a mountain range.
 }
 [sha:160baf04c6ac12c5b758da5aa609b15fa2085dc6]{
 -- Page 620, Offset 25917.
 -- 意思の疎通など出来ようのない、<瀑布|ばくふ>じみた力の<撓|たわ>み。
-A curving mass of power like a waterfall with no room for communication.
+// double check, I think I kinda messed this one up.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:1047]: I think the original is correct here 

Maybe something like: A curved mass of force like a waterfall; allowing no room for mutual understanding.

I don't really have anything great to offer as an alternative, but I do think the original's meaning is right

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 10]
```diff
@@ -552,7 +553,7 @@ It's true that this True Ancestor is exceptional. There's no record of victims f
 [sha:eea55b8994fe4a541fdb2d892bad19ecb2af9970]{
 -- Page 667, Offset 26045.
 -- 「うるさい！　そんな話なんてどうでもいい……！
-"Shut up! None of that stuff matters...!
+"Shut up! None of crap matters...!
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:556]: that

**Review by nolo16 — APPROVED** [review]
otsukare

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:4]: I disagree, "the hands on the clock" is kinda a set expression

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:442]: I like how terse "cold" is. I wouldn't go so far as to say its a set phrase, but it is a motif. The terseness makes it all the more unfamiliar or surprising sounding

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 1 above)
Comment (nolo16) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:4]: np then

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 4 above)
Comment (nolo16) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:442]: np with that

**Pull request merged**